recipes-example/template-dbus-service/template-dbus-service.bb

```diff
@@ -0,0 +1,35 @@
+# Copyright (C) 2019 Luxoft Sweden AB
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#
+# SPDX-License-Identifier: MPL-2.0
```

Comment:
we normally license recipes under MIT

Reply:
This has already been discussed and the conclusion was to use MPL-2.0 for the template and future D-Bus services. As is reflected in the template-dbus-service this recipe points to.
https://github.com/Pelagicore/template-dbus-service/blob/master/LICENSE